src/app/modules/ix-icon/ix-icon.component.ts

```diff
@@ -70,15 +70,24 @@ export class IxIconComponent extends MatIcon implements OnInit {
   }
 
   ngOnInit(): void {
-    if (this.iconName.startsWith('ix:')) {
-      this.svgIcon = this.iconName;
-    } else if (this.iconName.startsWith('mdi')) {
+    this.updateIcon(this.iconName);
+    super.ngOnInit();
+  }
+
+  ngOnChanges(): void {
+    this.updateIcon(this.iconName);
+  }
```

Comment:
Should parent `onChanges` be called as well?

Reply:
Parent class has no `ngOnChanges` method.